build/charts/antrea-windows/Chart.yaml

```diff
@@ -5,7 +5,7 @@ displayName: Antrea
 home: https://antrea.io/
 version: 0.0.0
 appVersion: latest
-kubeVersion: ">= 1.22.0-0"
+kubeVersion: ">= 1.19.0-0"
```

Comment:
Was there any reason why it was set to >=1.22.0 in the first place?

Reply:
Good question, I checked the history and found out it's actually a specific requirement for Windows because of host process feature is supported since v1.122: https://github.com/antrea-io/antrea/pull/6360#discussion_r1627349610
Let me close this PR.  A following question is about the minimal K8s version for Antrea, should we update the v1.19 to v1.22 because of Windows?

Reply:
I think we should mention the minimal versions for linux and windows separately.